core/server/src/test/java/alluxio/worker/block/BlockWorkerTest.java

```diff
@@ -294,6 +294,20 @@ public void getVolatileBlockMetaTest() throws Exception {
   }
 
   /**
+   * Tests the {@link BlockWorker#getBlockMeta(long)} method.
+   *
+   * @throws Exception if the getVolatileBlockMeta check fails
+   */
+  @Test
+  public void getBlockMetaTest() throws Exception {
+    long sessionId = mRandom.nextLong();
```

Comment:
I basically just mimic what has been done here, looks like all the other tests use a random number as the sessionId and blockId.